### ISSUE TITLE
Specifying version for Install ChefDK task fails

### ISSUE THREAD

**Issue by ehanlon** [chef-partners/vsts-chef#13]
Get the following error when I specify a version number in the task.

It looks like there is an extra `-` character added before the `-v`.

It also looks like it will not cause the task to fail since it's not in STDERR.

```
2018-04-04T17:46:19.9496832Z ##[section]Starting: Install ChefDK
2018-04-04T17:46:19.9561767Z ==============================================================================
2018-04-04T17:46:19.9574497Z Task         : Install ChefDK (Chef)
2018-04-04T17:46:19.9586325Z Description  : Install ChefDK on the agent
2018-04-04T17:46:19.9597827Z Version      : 1.4.21
2018-04-04T17:46:19.9616986Z Author       : Chef
2018-04-04T17:46:19.9633752Z Help         : [More information](https://github.com/chef-partners/vsts-chef/wiki/install-chefdk)
2018-04-04T17:46:19.9651056Z ==============================================================================
2018-04-04T17:46:20.8571557Z   % Total    % Received % Xferd  Average Speed   Time    Time     Time  Current
2018-04-04T17:46:20.8696775Z                                  Dload  Upload   Total   Spent    Left  Speed
2018-04-04T17:46:20.8719315Z 
2018-04-04T17:46:20.8754842Z   0     0    0     0    0     0      0      0 --:--:-- --:--:-- --:--:--     0
2018-04-04T17:46:20.8805286Z 100 23425  100 23425    0     0   151k      0 --:--:-- --:--:-- --:--:--  152k
2018-04-04T17:46:20.8848847Z /tmp/chef-install.sh: illegal option -- -
2018-04-04T17:46:20.8894995Z usage: /tmp/chef-install.sh [-P project] [-c release_channel] [-v version] [-f filename | -d download_dir] [-s install_strategy] [-l download_url_override] [-a checksum]
2018-04-04T17:46:21.0107601Z Installing Software: chefdk
2018-04-04T17:46:21.0122397Z ... Linux
2018-04-04T17:46:21.0269136Z [command]curl https://omnitruck.chef.io/install.sh --output /tmp/chef-install.sh
2018-04-04T17:46:21.0288844Z If installation fails you may need to enable sudo for the agent account with the NOPASSWD option
2018-04-04T17:46:21.0451680Z [command]bash /tmp/chef-install.sh -P chefdk -c stable --v 2.1
2018-04-04T17:46:21.0505023Z ##[section]Finishing: Install ChefDK
```

**Comment by russellseymour** [comment]
Thankyou @ehanlon for this
I have found the bug in the code. There will be a new release this week to fix this.

**Comment by russellseymour** [comment]
Fixed in release 1.4.40

https://github.com/chef-partners/vsts-chef/wiki/1.4.40
